lib/signature.dart

```diff
@@ -72,6 +74,8 @@ class SignatureState extends State<Signature> {
   //CLEAR POINTS AND REBUILD. CANVAS WILL BE BLANK
   void clear() {
     setState(() => _points.clear());
+    //NOTIFY OF CHANGE AFTER MOVEMENT IS DONE
+    widget.onChanged(_points);
```

Comment:
IMHO onChanged should be optional as not everyone needs to use it. That means this line should use null-safe operator (`?.`)